app/views/entitlements/index.html.erb

```diff
@@ -1,58 +1,54 @@
 <%- @page_title = 'Entitlements' -%>
 
-<div class="usa-grid usa-section usa-layout-docs">
-  <%= render 'shared/sidenav' %>
+<div class="usa-section main-section">
```

Comment:
Github isn't detecting that I'm just shifting indentation to the left for all the html files. All real changes to html pages only includes 4 lines -- removing the outer div and the side nav, and changing the classes for the main content div.

Reply:
Nevermind, I figured out how to view only the significant changes: 

https://github.com/deptofdefense/move.mil/pull/121/files?w=1

Reply:
@aileendds Oh, nice. I forgot about the `w=1` param.